.idea/aws.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove .idea folder

Reply:
+1 - Also, Update the .gitignore file to exclude .idea folder.